libs/barista-components/stacked-series-chart/src/stacked-series-chart.spec.ts

```diff
@@ -78,6 +79,11 @@ describe('DtStackedSeriesChart', () => {
     );
   }
 
+  /** Gets a specific stack at a specific position */
+  function getTrackByPositionWithinTrack(trackIndex: number): DebugElement {
```

Comment:
This naming could be improved to `getTrackByPosition` as we don't look for track inside track